### ISSUE TITLE
Background image is "Loading"

### ISSUE THREAD

**Issue by Cecilectomy** [clockworkpi/launcher#181]
The GameShell background image is the  "Loading" image (On HDMI it is actually a full size background image of an exploded gameshell). If and When the UI crashes due to various bugs and issues, it makes the unit look like it is stuck loading something when in actuality nothing is happening, you are just seeing the background image.

The background should be something unique, and the loading image should be shown in times of actual loading, and an error image should be shown for crashes, etc.

**Comment by Mihaylov93** [comment]
I think thats some technical limitation of how its shown.

feh is used for showing that at .xinitrc time
https://github.com/clockworkpi/launcher/blob/743c82fc2657231b5f9c908703ff873d053ff3c4/.xinitrc#L15

**Comment by Cecilectomy** [comment]
Right, but feh can be and is being used in numerous places to set the background to more appropriate images (bg image is not the most appropriate solution to begin with, but it's what being used)

I forgot about this issue, but at one point I had some example code that would change the image if the launcher encountered an unrecoverable issue to show an "error" image.

This is to update the code to display a more appropriate image than "loading" which causes a LOT of people to think their device is stuck loading whatever it is they last launched, but actually the launcher is just completely crashed.

If I ever get back around, I will be fixing this myself and creating a pull request.

**Comment by Mihaylov93** [comment]
It would be nice an error handler on python, to catch any exception and set the background. 

Or maybe a daemon.
